sugargame/event.py

```diff
@@ -170,10 +170,10 @@ def _keyevent(self, widget, event, type):
             self.__keystate[keycode] = type == pygame.KEYDOWN
             if type == pygame.KEYUP:
                 mod = self._keymods()
-            ukey = unichr(Gdk.keyval_to_unicode(event.keyval))
+            ukey = chr(Gdk.keyval_to_unicode(event.keyval))
             if ukey == '\000':
                 ukey = ''
-            evt = pygame.event.Event(type, key=keycode, unicode=ukey, mod=mod)
+            evt = pygame.event.Event(type, key=keycode, str=ukey, mod=mod)
```

Comment:
This should throw an error as [`pygame.event.Event`](https://www.pygame.org/docs/ref/event.html#pygame.event.Event) only takes two arguments, you can test this in your interpreter by calling the function with the needed arguments.

Reply:
I went through the documentation and also I checked many activities. This exists in all activities wrapped with sugargame and it seems to work fine. I'm not sure if there is a need to alter it. Also, I'm not fully aware of every argument in that line, So I don't know how make any changes to get it right (if required).

Reply:
> This should throw an error as pygame.event.Event only takes two arguments, you can test this in your interpreter by calling the function with the needed arguments.

`pygame.event.Event` allows two signatures
```python
Event(type, dict) -> EventType instance
Event(type, **attributes) -> EventType instance
```

I think @sourabhaa has used the second signature in this (`**kwargs`) which allows `n` number of key=value parameters to `pygame.event.Event`. imo, it shouldn't throw an error though.

Reply:
Okay, that might be the case but I'd ran it in an interpreter and got an error. I haven't tested the activity yet to know if it throws an error.